components/toast/index.css

```diff
@@ -30,54 +79,102 @@ governing permissions and limitations under the License.
+
+    .spectrum-Toast-closeButton {
+      &:focus-ring:not(:active) {
+        color: var(--mod-toast-negative-background-color-default, var(--spectrum-toast-negative-background-color-default));
+      }
+    }
+  }
+
+  &--info {
+      background-color: var(--mod-toast-informative-background-color-default, var(--spectrum-toast-informative-background-color-default));
+      color: var(--mod-toast-informative-background-color-default, var(--spectrum-toast-informative-background-color-default));
+
+    .spectrum-Toast-closeButton {
+      &:focus-ring:not(:active) {
+        color: var(--mod-toast-informative-background-color-default, var(--spectrum-toast-informative-background-color-default));
+      }
+    }
+  }
+
+  &--positive,
+    /* @deprecated */.spectrum-Toast--success {
+    background-color: var(--mod-toast-positive-background-color-default, var(--spectrum-toast-positive-background-color-default));
+    color: var(--mod-toast-positive-background-color-default, var(--spectrum-toast-positive-background-color-default));
+
+    .spectrum-Toast-closeButton {
+      &:focus-ring:not(:active) {
+        color: var(--mod-toast-positive-background-color-default, var(--spectrum-toast-positive-background-color-default));
+      }
+    }
+  }
 }
 
+/* TODO note what type icon is */
 .spectrum-Toast-typeIcon {
   flex-shrink: 0;
   flex-grow: 0;
 
-  margin-block: var(--spectrum-toast-icon-padding-y);
-  margin-inline-end: var(--spectrum-toast-neutral-icon-padding-right);
+  margin-block-start: var(--mod-toast-spacing-top-edge-to-icon, var(--spectrum-toast-spacing-top-edge-to-icon));
+  margin-inline-end: var(--mod-toast-spacing-icon-to-text, var(--spectrum-toast-spacing-icon-to-text));
   margin-inline-start: 0;
+
+  color: white;
 }
 
 .spectrum-Toast-content {
   flex: 1 1 auto;
   display: inline-block;
   box-sizing: border-box;
 
-  padding-block-start: var(--spectrum-toast-neutral-content-padding-top);
-  padding-inline-end: var(--spectrum-toast-neutral-content-padding-right);
-  padding-block-end: var(--spectrum-toast-content-padding-bottom);
+  padding-block-start: var(--mod-toast-spacing-top-edge-to-text, var(--spectrum-toast-spacing-top-edge-to-text));
+  padding-block-end: var(--mod-toast-spacing-bottom-edge-to-text, var(--spectrum-toast-spacing-bottom-edge-to-text));
+  padding-inline-end: var(--mod-toast-spacing-text-to-action-button-horizontal, var(--spectrum-toast-spacing-text-to-action-button-horizontal));
   padding-inline-start: 0;
 
-  font-size: var(--spectrum-toast-info-text-size);
-  font-weight: var(--spectrum-toast-info-text-font-weight);
-  line-height: var(--spectrum-toast-info-text-line-height);
+  line-height: var(--mod-toast-line-height, var(--spectrum-toast-line-height));
+
+  &:lang(js),
+  &:lang(zh),
+  &:lang(ko) {
+    line-height: var(--spectrum-toast-line-height-CJK, var(--mod-toast-line-height, var(--spectrum-toast-line-height)));
```

Comment:
I changed the token  naming format `--system-component-style-description-modifier` to try to be consistent. Are there any issues with this approach?

Reply:
I think this is fine. Thanks for the heads up!